notebooks/ullyses-walkthrough.ipynb

```diff
@@ -261,27 +316,43 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "# Create a variable to point to the file we just downloaded\n",
-    "spec_file = Path('./hlsp_ullyses_hst_cos-stis_sk-67d195_uv_dr6_preview-spec.fits')\n",
-    "\n",
-    "# Use Astropy.fits to open the fits file and create a header data unit (HDU)\n",
-    "# More info on fits files and HDUs can be found at: https://docs.astropy.org/en/stable/io/fits/\n",
-    "spec_hdu = fits.open(spec_file)\n",
+    "targets = ['XX CHA', 'SK -67 167']\n",
     "\n",
-    "# Print out some information about the format and data in this HDU\n",
-    "print(spec_hdu.info())"
+    "for targ in targets:\n",
+    "    ull_targ = match_aliases(targ).lower()\n",
+    "    print(ull_targ)\n",
+    "    # Create a variable to point to the file we just downloaded\n",
+    "    spec_filename = glob.glob(f'*{ull_targ}*_preview-spec.fits')[0]\n",
+    "    spec_file = Path(spec_filename)\n",
+    "    \n",
+    "    # Use Astropy.fits to open the fits file and create a header data unit (HDU)\n",
+    "    # More info on fits files and HDUs can be found at: https://docs.astropy.org/en/stable/io/fits/\n",
+    "    with fits.open(spec_file) as spec_hdu: \n",
+    "        # Print out some information about the format and data in this HDU\n",
+    "        print(spec_hdu.info())\n",
+    "        \n",
+    "    print(f\"**Filename components: {spec_filename.split('_')}\")\n",
+    "    print()"
    ]
   },
   {
    "cell_type": "markdown",
    "metadata": {},
    "source": [
     "\\\n",
-    "From the filename, we can see:\n",
+    "From the filename, `hlsp_ullyses_hst_cos-stis_v-xx-cha_uv-opt_dr7_preview-spec.fits`, we can see:\n",
     "- the telescope used is HST\n",
-    "- the instrument used are COS and STIS\n",
-    "- the optical element is UV (in this case, many optical elements made up this data product)\n",
-    "- this file was created for data release 6\n",
+    "- the instruments used are COS and STIS\n",
+    "- the target observed is V-XX-CHA\n",
+    "- the optical range is UV and Optical (in this case, many optical elements made up this data product)\n",
```

Comment:
I am not 100% sure what this means, can you explain further? "(in this case, many optical elements made up this data product)"

Reply:
Also I think this (and the target below it) should say "wavelength range" rather than "optical range"

Reply:
@jotaylor I think this was left over from the old naming scheme for the preview spec files for DR6 where the gratings were listed instead of "uv" or "opt". I've just removed that comment since it's a little confusing now.